api/UIEvent.json

```diff
@@ -50,18 +50,384 @@
           "deprecated": false
         }
       },
+      "UIEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/UIEvent/UIEvent",
+          "description": "<code>UIEvent()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "11"
+            },
+            "firefox_android": {
+              "version_added": "14"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "cancelBubble": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/UIEvent/cancelBubble",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        },
```

Comment:
Seeing `null` for all mobile versions here https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/cancelBubble

Reply:
In the case of this one specifically, I referenced the [base page](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent#Browser_compatibility). I usually reference the sub-feature pages unless the base page contains more complete info.

Reply:
I referenced the [base page](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent) because it seemed to contain more complete info.